src/codegen/intermediate_representation.c

```diff
@@ -225,6 +225,18 @@ void ir_femit_instruction
     fprint(file, "%33not %34%%%u", inst->operand->id);
     break;
 
+  case IR_ZERO_EXTEND:
+    fprint(file, "%33z.ext %34%%%u", inst->operand->id);
+    break;
+
+  case IR_SIGN_EXTEND:
+    fprint(file, "%s.ext %34%%%u", inst->operand->id);
```

Comment:
`%s`? :eyes: 

I can see you still can’t run the format check plugin on your system...

Reply:
Hahaha. Must be a copy paste error :P

Fixed in https://github.com/LensPlaysGames/Intercept/pull/59/commits/2246f711a0d5268f59764797be72ec6d189b1ed6